src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,20 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private static final int MACHINE_NUMBER = 3;
+
+    public List<Bulldozer> get() {
+        Bulldozer bulldozer = new Bulldozer();
+        Bulldozer bulldozer1 = new Bulldozer();
+        Bulldozer bulldozer2 = new Bulldozer();
+        List<Bulldozer> list = new ArrayList<>();
+        list.add(bulldozer);

```

Comment:
Instead of doing some identical operations, you can use loops.
Here you can use your "MACHINE_NUMBER" constant.